tests/test_dragon_launcher.py

```diff
@@ -220,7 +242,7 @@ def test_secure_socket_authenticator_setup(
         assert authenticator.num_configure_curves > 0
         # ensure authenticator was started
         assert authenticator.num_starts > 0
-        assert authenticator.context == context
+        assert authenticator.context == zmq.Context.instance()
```

Comment:
and here

Reply:
Looks like there are a few more calls across the test suite. If it was intended, feel free to resolve all of these comments!

Reply:
Ok for tests, resolving!